docs/en/ingest-management/commands.asciidoc

```diff
@@ -72,19 +173,48 @@ This flag is helpful when using automation software or scripted deployments.
 Show help for the `enroll` command.
 
 `--insecure`::
-Allow an insecure connection to {kib}. When this flag is specified, API keys are
-sent in clear text. We strongly recommend that you use a secure connection.
+Allow insecure connections to {fleet-server}. When this flag is specified, API
+keys are sent in clear text. We strongly recommend that you use a secure
+connection.
+
+//QUESTION: Is this true? Are the keys really sent in clear text, or does this
+//have to do with accepting auto-generated certificates?
```

Comment:
When communicating with a HTTP server `--insecure` is required and "yes" the content is sent in clear text. Then connecting to HTTPS server with `--insecure` because the certificate chain cannot be verified, that means the content is encrypted (not clear-text) but the certificate cannot be verified.